libV2/cache/backend/redis.js

```diff
@@ -43,26 +44,19 @@ class RedisCache {
     }
 
     async getKey(key) {
-        try {
-            return this._redis.get(key);
-        } catch (error) {
-            moduleLogger
-                .with({ method: 'getKey' })
-                .error('error fetching key from redis', { key });
-            throw error;
-        }
+        return moduleLogger
+            .with({ method: 'getKey' })
+            .logAsyncError(() => this._redis.get(key),
+                'error fetching key from redis', { key });
```

Comment:
I am not sure where `{ key }` is used, the log async error method only has two parameters.

Reply:
ah, it's supposed to be passed along like the other methods